contracts/token/ERC721.cairo

```diff
@@ -111,6 +118,16 @@ func isApprovedForAll{
     return (is_approved)
 end
 
+@view
+func tokenURI{
+        syscall_ptr: felt*, 
+        pedersen_ptr: HashBuiltin*, 
+        range_check_ptr
+    }(token_id: Uint256) -> (token_uri: felt):
```

Comment:
I think we need to be consistent and name parameters in camel case in external functions

Reply:
I updated the parameters for both external and view methods. We can update the other contracts in another PR. I left the `data_len` param for `safeTransferFrom`; as, Cairo expects the snake case for array lengths ie `_len`

Reply:
of course, that's a valid exception